content/body/textbox.php

```diff
@@ -227,20 +237,44 @@
 {
   "replaceHtmlRules": {},
   "steps": [
-
     {
-      "label": "Place an aria-describedby for instructions",
-      "highlight": "%INLINE%charcount-example ||| aria-describedby"
+      "label": "Set the 'maxlength' attribute of the textarea element.",
+      "highlight": "%INLINE%charcount-example ||| maxlength"
     },
     {
-      "label": "Code the instructions for this component.",
-      "highlight": "%INLINE%enable-character-count__global ||| id=\"character-count__desc\"",
-      "notes": "This is the target of the <code>aria-describedby</code> in the previous step."
+      "label": "Set hasCharacterCount for the textarea element.",
```

Comment:
Should we use `data-has-character-count` instead of `hasCharacterCount`, since that's what's in the code?

```suggestion
      "label": "Set the custom data-has-character-count attribute for the textarea element.",
```